modules/media/ajax/FileUpload.php

```diff
@@ -108,13 +108,13 @@ function uploadFile()
     $pscid      = isset($_POST['pscid']) ? $_POST['pscid'] : null;
     $visit      = isset($_POST['visitLabel']) ? $_POST['visitLabel'] : null;
     $instrument = isset($_POST['instrument']) ? $_POST['instrument'] : null;
-    $site       = isset($_POST['forSite']) ? $_POST['forSite'] : null;
     $dateTaken  = isset($_POST['dateTaken']) ? $_POST['dateTaken'] : null;
     $comments   = isset($_POST['comments']) ? $_POST['comments'] : null;
     $language   = isset($_POST['language']) ? $_POST['language'] : null;
 
     // If required fields are not set, show an error
-    if (!isset($_FILES) || !isset($pscid) || !isset($visit) || !isset($site)) {
+    if (!isset($_FILES) || !isset($pscid) || !isset($visit) ||
```

Comment:
Just FYI you can pass multiple argument to isset

```php
if (!isset($arg1, $arg2, ...))
```

Reply:
love it

Reply:
@johnsaigle looks like that should fix the phpcs errors as well